namespace cpp global.protocol
namespace as3 com.dreamfire.sanguo.data.definition.protocol.global

enum KickoutActorType {
  RELOGIN = 0,            // 重复登陆
  UNKNOWN,                // 未知

  MAX,
}

